woocommerce/payment-gateway/payment-tokens/class-sv-wc-payment-gateway-payment-token.php

```diff
@@ -136,6 +137,32 @@ public function get_id() {
 	}
 
 
+	/**
+	 * Gets the ID of the user associated with the token.
+	 *
+	 * @since 5.6.0-dev.1
+	 *
+	 * @return int
+	 */
+	public function get_user_id() {
```

Comment:
@unfulvio these should be two separate commits to be atomic the way we have been briefly discussing